crates/ruff_python_parser/resources/valid/expressions/named.py

```diff
@@ -1,12 +1,8 @@
-if x := 1:
-    pass
-
-(x := 5)
-(x := (y * z))
-{x := 1}
-[x := 1]
```

Comment:
Are the list and set like tests covered in other tests? Because they seem to be valid syntax? 

I think it's okay to repeat some test cases.

Reply:
Yes, they're covered in the respective container test cases.